source/PlatformTracker.hx

```diff
@@ -0,0 +1,36 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.input.keyboard.FlxKey;
+
+/**
+ * ...
+ * @author Jess Geiger
+ */
+
+ class PlatformTracker extends Player
+ {
+    public var trackedOffsetX:Float;
+    public var baseObj:FlxObject;
+
+    public function new(?obj:FlxObject, ?offsetX:Float = 0)
+	{
+        super(); 
+        trackedOffsetX = offsetX;  
+        baseObj = obj;    
+    }
+
+    public function returnOffset():Float
```

Comment:
Or you could get rid of it entirely, since baseObj is a public variable and can be accessed without this function.